src/caret_analyze/record/frequency.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Tuple
+
+from .column import ColumnValue
+from .interface import RecordsInterface
+from .record_factory import RecordFactory, RecordsFactory
+
+
+class Frequency:
+
+    def __init__(
+        self,
+        records: RecordsInterface,
+        target_column: Optional[str] = None
+    ) -> None:
+        """
+        Constructor.
+
+        Parameters
+        ----------
+        records : RecordsInterface
+            records to calculate frequency.
+        target_column : Optional[str], optional
+            Column name of timestamps used in the calculation, by default None
+            If None, the first column of records is selected.
+
+        """
+        self._target_column = target_column or records.columns[0]
+        self._target_timestamps: List[int] = []
+        for record in records:
+            try:
+                timestamp = record.get(self._target_column)
+            except (IndexError, KeyError):
+                continue
+            if timestamp is not None:
+                self._target_timestamps.append(timestamp)
```

Comment:
How about this one?

```suggestion
            if self._target_column in record.columns:
                timestamp = record.get(self._target_column)
                self._target_timestamps.append(timestamp)
```

My concerns are:
- An already added timestamp may be added into `self._target_timestamps` again.
- An exception may be too heavy to loop for `record`.

Reply:
Fixed: https://github.com/tier4/CARET_analyze/pull/116/commits/79bda3aa5a84afb089f98915f40810ddeb366ba5